.buildkite/scripts/periodic.trigger.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "steps:"
+
+source .buildkite/scripts/branches.sh
+
+for BRANCH in "${BRANCHES[@]}"; do
```

Comment:
How are we handling "last good commit" for our periodic builds? We should be triggering these builds using the last green commit from intake, not necessarily HEAD of the target branch.

Reply:
Okay, I just added that using what you did for serverless. I'll make sure intake gets running first, so that this won't just be broken in the meantime